tests/test_utilities.py

```diff
@@ -36,44 +37,48 @@ def lsst_btk_counts(request):
 )
 def lsst_btk_sky_level(request):
     survey, filt = request.param
-    expected = int(MEAN_SKY_LEVEL[f"{survey}_{filt}"])
+    expected = int(WLD_SKY_LEVEL[f"{survey}_{filt}"])
     return survey, filt, expected
 
 
 def test_mag2counts_str(lsst_btk_counts):
     survey, filt, expected = lsst_btk_counts
-    assert mag2counts(24, survey, filt).value == expected
+    np.testing.assert_allclose(mag2counts(24, survey, filt).value, expected, rtol=1e-1)
 
 
 def test_mag2counts_filter_str(lsst_btk_counts):
     survey, filt, expected = lsst_btk_counts
     survey_inst = get_survey(survey)
-    assert mag2counts(24, survey_inst, filt).value == expected
+    np.testing.assert_allclose(
+        mag2counts(24, survey_inst, filt).value, expected, rtol=1e-1
+    )
 
 
 def test_mag2counts_filter_instance(lsst_btk_counts):
     survey, filt, expected = lsst_btk_counts
     survey_inst = get_survey(survey)
     filt_inst = survey_inst.get_filter(filt)
-    assert mag2counts(24, survey_inst, filt_inst).value == expected
+    np.testing.assert_allclose(
+        mag2counts(24, survey, filt_inst).value, expected, rtol=1e-1
+    )
 
 
 def test_mean_sky_level_str(lsst_btk_sky_level):
     survey, filt, expected = lsst_btk_sky_level
     sky_level = mean_sky_level(survey, filt).value
-    assert int(sky_level) == expected
+    np.testing.assert_allclose(int(sky_level), expected, rtol=1e-1)
```

Comment:
For all the mean_sky_level tests, if you are testing relative tolerance, then the cast to integer should be removed from the test.

Reply:
sounds good, I just removed them.